natives.json

```diff
@@ -23657,7 +23657,7 @@
 			"gta_jhash": "0x25D569EB"
 		},
 		"0x13430D3D5A45F14B": {
-			"name": "_0x13430D3D5A45F14B",
+			"name": "DRAW_LOW_QUALITY_PHOTO",
```

Comment:
I can see why you chose this name but it's not correct because (1) this name does not match the jhash and (2) the actual name starts with ``GET_S*``, very likely ``GET_STATUS_OF_*``. So I'd name it ``_GET_STATUS_OF_DRAW_LOW_QUALITY_PHOTO`` or something similar, unless you can find the real name. I tried a few names and ran a quick bruteforce task but to no avail.